memphis/memphis.py

```diff
@@ -484,6 +495,28 @@ def validateJsonSchema(self, message):
         except Exception as e:
             raise MemphisSchemaError("Schema validation has failed: " + str(e))
 
+    def validateGraphQL(self, message):
```

Comment:
writing it here since I can't comment on lines you didn't edit
in line 484 you call schema_obj = json.loads(schema), this function should be called when a schema update is coming and not for each message